.babelrc

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/224420103-e3361857-6b65-4528-bc89-5cc0e5bc229e.png)

- [ ] It would be nice if you remove your Live Demo section because you don't have any live link for your application 🤔 Also to make your readme file professional 👍 Once you would have a live link then you can add the Live Demo section with the live link of your application 😊

Reply:
Thank you so much @HAMMAS-SALEEM 😇

Reply:
Thank you @HAMMAS-SALEEM We have now rectified that